mahta/CSS/CSS-size-units/styles.css

```diff
@@ -0,0 +1,34 @@
+.wrapperA {
+    width: 300px;
+    height: 100px;
+    border: 5px solid rebeccapurple;  
+  }
+  
+.wrapperB {
+    width: 40vw;
+    height: 200px;
+    border: 5px solid salmon;
+}
+
+.wrapperC {
+    font-size: 100em;
+    width: auto;
+    height: auto;
+    border: 5px solid aqua;
+}
+.wrapperC:root {
+    font-size: 30rem;
+}
+
+.wrapperA > .wrapperD {
```

Comment:
You don't need `.wrapperA >` section of the selector.

Reply:
I'm not certain about this comment but I did what I thought I must. I'm waiting for further instructions.